lib/src/io.cpp

```diff
@@ -436,10 +437,10 @@ stream_offsets findoffsets( dl::stream& file) noexcept (false) {
     return ofs;
 }
 
-std::vector< std::pair< std::string, long long > >
+std::map< dl::ident, std::vector< long long > >
```

Comment:
Commit 48ea8d2c36b40d8bcb3dfbf83634eb1081a1f11b doesn't compile without chnages from b0f208e65714bdb282b83c62ee4b8c24bebdbbb3

Reply:
What, no? I just ran `git rebase -i --exec="cd build && make -j6 && ctest --output-on-failure" 1483e32` without any issues

Reply:
Why 1483e32 when the problem is in 48ea8d2?

Reply:
That means, pick every commit _from_ 1483e32 to HEAD and run `cd build && make -j6 && ctest --output-on-failure`.